docker/package/systemd/tezos-node.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+After=network.target
+Description=Tezos node
+
+[Service]
+EnvironmentFile=/etc/defaults/tezos-node.conf
+Environment="TEZOS_CLIENT_UNSAFE_DISABLE_DISCLAIMER=Y"
+Environment="TEZOS_LOG=* -> info"
+ExecStart=/usr/bin/tezos-node-start
+StateDirectory=tezos
+User=tezos
```

Comment:
Apparently some service files have 2 `\n` in the end, some files have 1 and this file has 0.